ote/src/clj/ote/services/pre_notices/operator.clj

```diff
@@ -36,10 +53,19 @@
     db user (::t-operator/id notice)
     (fn []
       (tx/with-transaction db
-        (let [n (-> notice
-                    (modification/with-modification-fields ::transit/id user))]
-          (log/debug "Save notice: " n)
-          (specql/upsert! db ::transit/pre-notice n))))))
+         (let [attachment-names (:attachments notice)
+               n (-> notice
+                     (dissoc :attachments)
+                     (modification/with-modification-fields ::transit/id user))
+               saved-notice (specql/upsert! db ::transit/pre-notice n)
+               notice-id (::transit/id saved-notice)
+               attachments (parse-attachments attachment-names db user)
+               ments (mapv #(assoc % ::transit/pre-notice-id notice-id) attachments)]
+           (log/debug "Save notice: " saved-notice)
+           (log/debug "Attachments: " attachments)
+           (doall
```

Comment:
I presume this is done for the side-effect, not the return value.
It is clearer to use `doseq` instead of `doall` and `mapv`.

Instead of upserting, it's better to `update!` (we know that the attachment exists, we are just changing one field).

Reply:
what are `ments`?